doc/api/fs.md

```diff
@@ -1555,6 +1559,9 @@ If `fd` points to a character device that only supports blocking reads
 available. This can prevent the process from exiting and the stream from
 closing naturally.
 
+`fs` allows providing alternative `fs` implementation. It needs to provide the
+`open`, `read` and `close` methods.
```

Comment:
According to the [style guide:](https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md)
```suggestion
`open()`, `read()` and `close()` methods.
```